djangocms_text_ckeditor/cms_plugins.py

```diff
@@ -331,6 +320,14 @@ def __init__(self, *args, **kwargs):
 
         return TextPluginForm
 
+    @staticmethod
+    def _create_ghost_plugin(placeholder, plugin):
```

Comment:
Since the _create_ghost_plugin is a method specifically for cms 4, shouldn't the usage be put behind a version check as well?

Reply:
Thanks, @vinitkumar, for going through the code. 

This function actually runs both in v3 and v4. For v3 it just calls `plugin.save()`. For v4 it uses the new placeholder api: `palceholder.add_plugin(...)`. 

Since in v3 this api is not present the if statement effectively checks for version 4 (literally for the availability of the placeholder api)